backend/webapi/Models/Banner.cs

```diff
@@ -0,0 +1,18 @@
+namespace Pidp.Models;
+
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+using NodaTime;
+
+[Table(nameof(Banner))]
+public class Banner : BaseAuditable
+{
+    [Key]
+    public int Id { get; set; }
+    public string Component { get; set; } = string.Empty;
+    public string Status { get; set; } = string.Empty;
```

Comment:
I think it would be useful to have a `BannerStatus` enum rather than a string here, even if it only has one value currently.  Take a look at `EndorsementRequestStatus` for what that might look like.

Reply:
fixed.